packages/react-utilities/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "extends": "../../tsconfig.base.json",
```

Comment:
@Hotell  I think that you said this should not be done yet and that you would automate the conversion to tsconfig.base after you're doing with `react-menu` right ?

Reply:
@ling1726  💯 yesir! thx